src/App.tsx

```diff
@@ -681,6 +768,29 @@ const MapDisplay = (
                       (100 / (zoomLevel || 100))
                     )
 
+                    // Road Click
+                    let road = -1
```

Comment:
Maybe better variable name could be `clickedRoadQuadrant`. Also you could use undefined as the value when not clicked instead of -1, I find that to be a bit clearer.